Add audit logs retention section (#235)
permitio/docs: docs/how-to/use-audit-logs/types-and-filtering.mdx

  <source src={viewingAuditLogs} />
</video>

:::note Audit Log retention

> We periodically delete old audit logs to maintain system efficiency. If you need access to older logs, please reach out to us via Slack or email us at [help@permit.io](mailto:help@permit.io). We also offer plans for extended audit log retention. For more details, contact us directly.

Reviewer: ```suggestion
We periodically delete old audit logs to maintain system efficiency. If you need access to older logs or would like to extend log retention periods for your account, please reach out to us via [Slack](io.permit.io/slack) or email us at [help@permit.io](mailto:help@permit.io).
```
